### PR TITLE
Added project for Daniel^2

### PR DESCRIPTION
**Pull request by fadoeng**

### CODE REVIEW

File: Lean/Students_projects/Daniel^2/Daniel^2_FTA_Corollary4.pdf
Comment (matematiflo) [inline Lean/Students_projects/Daniel^2/Daniel^2_FTA_Corollary4.pdf:1]: This needs to be reviewed. As stated, the statement does not quite make sense. Is the conclusion the fact that the two commuting operators have a common eigenvector? If so, it is not stated in this way. Please review and fix this, thanks!

**Review by matematiflo — CHANGES_REQUESTED** [review]
Please see the separate review of the pdf file

**Comment by fadoeng** [comment]
Added the statement in the .pdf file

**Comment by fadoeng** [comment]
Is this possible to merge? I'm kinda confused with GitHub now.

**Pull request merged**